sys/parsoid.js

```diff
@@ -213,8 +213,11 @@ class ParsoidService {
         };
     }
 
-    getBucketURI(rp, format, tid, useKeyRevValue) {
-        const bucket = useKeyRevValue ? 'key_rev_value' : this.options.bucket_type;
+    getBucketURI(rp, format, tid, forceKeyRevValue, useNewBucket) {
```

Comment:
Minor niggle: It might be more readable & require less changes across the module to introduce a second "getOldBucketURI" method, and then call that only in the few places we need to access it.